### ISSUE TITLE
Problems with debian packages.

### ISSUE THREAD

**Issue by linsomniac** [influxdata/telegraf#264]
Thank you very much for the debian packages. However, there are a few things I think need to be changed:

The package installs things into opt sub-directories of other directories: /etc/opt/telegraf. These are just the wrong places for them. Really the config file should go into /etc/telegraf.conf, or possibly /etc/telegraf/telegraf.conf (though since there is only one file, it doesn't make sense to make a directory.

The logrotate file needs to include "copytruncate", since it doesn't signal telegraf to re-open the log-file, otherwise it leaves no log file and the current telegraf instance keeps writing the existing (removed) file until it is restarted.

The logrotate file should probably also have "dateext" to match influxdb?

Typically, packages don't put things into /opt, the packages manage the installed files so you would just put the daemon into /usr/sbin, the init scripts either directly into /etc/init or /etc/init.d, or into /usr/share and copy/link them into /etc/init in the post install.

The package also lists a file "/.". That probably doesn't hurt anything but also could be cleaned up.

**Comment by sparrc** [comment]
Can't say I disagree with these points, I believe `opt` is actually the technical standard, but in practice I find usage of `opt` kind of pointless and ignored by almost everyone.

@linsomniac I'll merge the PR you sent up, and then talk to others here at Influx and see if we can eliminate usage of `opt` as well

**Comment by linsomniac** [comment]
See the discussion on influxdb for my similar report and PR:

https://github.com/influxdb/influxdb/issues/4446

**Comment by sparrc** [comment]
Thanks, can you send up another PR to fix the rest of the points you made?

**Comment by linsomniac** [comment]
Probably.  The package.sh script is a slightly unusual way of doing it (instead of using a debian/ directory), but I can probably figure it out.  From the influxdb side of things, it sounds like it is something they are open to, so it sounds like I could have some traction if I made the changes I proposed.  Does the proposal sound good to you?

**Comment by sparrc** [comment]
yes, except please link the binary in `/usr/bin` instead of putting it in `/usr/sbin`

**Comment by linsomniac** [comment]
Please re-open.

**Comment by amoghe** [comment]
Since this bug tracks issues with installing from a debian pacakge, I'll add this here:

On installation, I see this:
```
/var/lib/dpkg/info/telegraf.postinst: 9: /var/lib/dpkg/info/telegraf.postinst: [[: not found
```

Probably due to bash syntax in a file that is executed under `sh`? 

Happy to open a new issue to track this if needed.

Details of downloaded deb file:
```
root@4ec6b49cb4ce:/opt/influxdb# md5sum /tmp/telegraf_0.1.9_amd64.deb 
3ed63f3b83a7561ce73afd6114e10fe8  /tmp/telegraf_0.1.9_amd64.deb
```

**Comment by sparrc** [comment]
That issue has been fixed, it will be available in the next release

**Comment by rossmcdonald** [comment]
Fixed with #497
